en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -97,97 +99,180 @@ In this tutorial, let's design the REST API by updating the low-code diagram as
+
+       1. In the **Action** pane, click **covid19Ep**.
+      
+       2. Click **getStatusByCountry**.
+      
+       3. In the action statement, double-click **`getStatusByCountryResponse`** variable name to edit it. Replace the default value with `statusByCountry`.
+      
+       4. In the **Parameters** tab, check whether the **country** check box is selected. If not, select it.
+      
+          This adds `country` as a query parameter in the connector configuration to allow the REST API to fetch COVID-19 statistics specific to a given country.
+        
+       5. Click **Save**.
+        
+   5. To extract the response of the **COVID-19** connector, add a variable by following these sub-steps.
+   
+       1. Click the last **+** in the low-code diagram.
+      
+       2. Click **Variable**.
+      
+          In the **Variable** pane, configure the variable as follows:
+      
+           1. On the variable statement, click **`var`**, and click **int** on the list that appears below in the **Suggestions** tab. Here, you are changing the variable type from `var` to `int`.
+         
+           2. To edit the variable name, double-click **`variable`**. Then enter `totalCases` as the variable name.
+         
+           3. Click  **`<add-expression>`** and click **statusByCountry** in the list that appears below in the **Suggestions** tab. Here, you are selecting the **statusByCountry** query parameter you previously added to derive information related to a specific country.
+         
+              Update the expression further as follows:
+         
+               1. In the list of second-level suggestions that appear in the **Suggestions** tab under **statusByCountry**, click **cases decimal**. Here, you are specifying that you want the total number of cases per country. The connector returns this information in the `decimal` format.
+               
+               2. To cast the total number of cases per country in `int` format instead of in the `decimal` format, follow these sub-steps:
+               
+                  1. Click **Expressions**. Click **<type>Es** under the **Type Cast** category.
+                   
+                  2. On the variable statement, click **`<<add-type>>`**.
+                   
+                  3. Click **Suggestions**, and click **int** in the list of suggestions.
+                   
+           4. Click **Save**.
+          
+                The statement should now be as follows:
+       
+                `<int>statusByCountry.cases`
+         
+   6. The REST API needs to further process the COVID-19 cases per country you derived to present the number of cases per one million people in the given country's population. Therefore, let's derive population statistics per country via the **World Bank** as follows:
+   
+       1. Click the last **+** in the low-code diagram.
+      
+       2. In the **Connectors** pane, click **World Bank by ballerinax**.
```

Comment:
We would also need to mention the following:
Search for **World Bank**, and select the **World Bank by ballerinax** connector when it appears in the search results.